### ISSUE TITLE
VM creation error message should be meaningful

### ISSUE THREAD

**Issue by bingoarun** [Azure/azure-sdk-for-python#11416]
- **Package Name**: azure-mgmt-compute
- **Package Version**: 9.0.0 (also tried on 12.0.0, same error)
- **Operating System**: Ubuntu
- **Python Version**: Python 2.7.15+

**Describe the bug**
Can't create VM, it errors at 
```
...
  File "/home/arun/projects/newvirtualenv/local/lib/python2.7/site-packages/azure/mgmt/compute/v2019_03_01/operations/virtual_machines_operations.py", line 298, in create_or_update
    **operation_config
  File "/home/arun/projects/newvirtualenv/local/lib/python2.7/site-packages/azure/mgmt/compute/v2019_03_01/operations/virtual_machines_operations.py", line 252, in _create_or_update_initial
    raise exp
msrestazure.azure_exceptions.CloudError: <exception str() failed>
```
Later identified that is a quota issue. I would like to see in the error message that it is a quota issue. 

**To Reproduce**
1. Create a VM using azure-mgmt-compute when you are out of quota

**Expected behavior**
The error should show that there is a quota issue

**Comment by lmazuel** [comment]
Hi @bingoarun 
This string exception error was fixed in the underlying runtime "msrestazure 0.6.3". Could you update to that version and confirm that you get a more interesting message?
Thank you

**Comment by Drewm3** [comment]
It looks like this issue has been resolved. Please reopen the issue if the correct error message is not seen.
